### DISCUSSION TITLE
No apps visible after logging in using SSO

### DISCUSSION THREAD

**Discussion by r3econ** [argoproj/argo-cd#8434] [Q&A]
I configured GitHub SSO using the official [guide](https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#dex) and I am able to log in to ArgoCD dashboard. The problem is that when I log in, I don't see any apps in the dashboard (I have 3 apps deployed using Argo in the cluster).

What am I missing? Do I have to give permission to the new user to see the apps? How would I go about it?
Thanks for the help!

I'm using ArgoCD `v2.2.2+03b17e0`

![Screenshot 2022-02-08 at 17 33 29](https://user-images.githubusercontent.com/505054/153033457-b4a4828a-4a7d-436a-a303-90a02d16ec29.png)

**Comment by chetan-rns** [comment]
@r3econ For configuring RBAC roles for users please check https://argo-cd.readthedocs.io/en/stable/operator-manual/rbac/
